client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -274,13 +279,69 @@ class CampaignAdminList extends SilverStripeComponent {
         </div>
       );
     } else {
-      preview = <Preview itemLinks={itemLinks} itemId={itemId} onBack={this.handleCloseItem} />;
+      preview = (
+        <Preview
+          itemLinks={itemLinks}
+          itemId={itemId}
+          onBack={this.handleCloseItem}
+          moreActions={this.getMoreActions()}
+        />
+      );
     }
 
     return preview;
   }
 
   /**
+   * @return {array}
+   */
+  getMoreActions() {
+    const selectedItem = this.getSelectedItem();
+
+    if (!selectedItem) {
+      return null;
+    }
+
+    const requiredByNum =
+      selectedItem._links.referenced_by &&
+      selectedItem._links.referenced_by.length || 0;
+    const unremoveableInfoText = requiredByNum < 2 ?
+      i18n._t(
+        'CampaignAdmin.UNREMOVEABLE_INFO_SINGULAR',
+        'Required by %s item, and cannot be removed directly.'
+      ) :
+      i18n._t(
+        'CampaignAdmin.UNREMOVEABLE_INFO_PLURAL',
+        'Required by %s items, and cannot be removed directly.'
+      );
+    const removeAction = selectedItem.Added === 'explicitly' ?
+      (
+        <button key="remove_action"
+          className="btn btn-secondary action"
+          onClick={this.handleRemoveItem}
+        >
+          Remove
+        </button>
+      ) :
+      (
+        <p key="unremoveable_info" className="alert alert-info campaign-admin__unremoveable-item">
+          <span className="font-icon-link"></span>
+          {i18n.sprintf(unremoveableInfoText, requiredByNum)}
```

Comment:
Please use i18n.inject() with named attributes instead of i18n.sprintf(), so we don't have positional issues later on :)